### DISCUSSION TITLE
where is the _book directory?

### DISCUSSION THREAD

**Discussion by shaikhFaris** [sumn2u/learn-javascript#215] [General]
I am not able to find the _book directory .

**Comment by shaikhFaris** [comment]
Update: I got to know that it appears after executing npm start.

**Reply by sumn2u** [reply]
True. It appears after running the `npm start` command. The `_book` is the compiled version of the work. If you are planning to make modifications, you can do so by navigating to the respective language directory and making changes there. For example, if I want to add a new chapter to the English directory, I will go inside the `en` folder and add the new chapter there. 



    learn-javascript
    ├── _book                   # compiled version of contents
    ├── _layouts                # layout for rendering book/website
    ├── en                      # content of book in english language
    │     └──  new chapter             #  new chapter in English
    ├── fr                      # content of book in french language
    ├── np                      # content of book in nepali language
    ├── tr                      # content of book in turkish language
    ├── LANGS.md                # supported languages
    ├── books                   # books in pdf and epub formats with translation
    ├── book.json               # configuration for book
    └── ...

**Comment by shaikhFaris** [comment]
Which framework/tool are u using for your website? How can I add animations to your home page?

**Reply by sumn2u** [reply]
We are using [Honkit](https://github.com/honkit/honkit). While I'm not sure what kind of animations you're planning to add, please note that to include animations, we will need to make modifications to the (theme)[[gitbook-plugin-theme-creative](https://github.com/sumn2u/gitbook-plugin-theme-creative)].
